project-canvas/src/components/backlog-view/BacklogView.tsx

```diff
@@ -188,10 +189,12 @@ export function BacklogView() {
             }}
           >
             {searchedissuesWrappers.get("Backlog") && (
-              <DraggableIssuesWrapper
-                id="Backlog"
-                issues={searchedissuesWrappers.get("Backlog")!.issues}
-              />
+              <Box mr="sm">
```

Comment:
i prefer doing it on the outermost component, so that if you add other sub components that aren't draggableIssueWrappers, they also have that same margin